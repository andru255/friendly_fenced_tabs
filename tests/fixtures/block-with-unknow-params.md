```my_unknow_param='foo' hl_lines='2 3' foo='bar'
my example of block
lorem ipsum
lorem ipsum
```