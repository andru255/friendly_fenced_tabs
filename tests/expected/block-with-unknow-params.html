<pre>
    <code>
        my example of block
        lorem ipsum
        lorem ipsum
    </code>
</pre>